modules/task/task.go

```diff
@@ -56,6 +46,12 @@ func GlobalBackend() Backend {
 	return _globalBackend
 }
 
+func globalBackendStatsClient() *statsClient {
```

Comment:
Ya it should be eventually, but to make the modules/task package not use globals, there needs to be a bit more separation. Really what you need is something more like `Frontend` and `Backend`, and `Encoder` needs to be split up into `Encoder` and `Decoder`. Right now though, there's no such concept, so stats for what would be on the `Frontend` are accessed in the same way as the `Backend`, ie the `Enqueue` function. If we get this PR in, when we look to get rid of the globals in modules/task, this will make it easier.